tests/i18n/tests.py

```diff
@@ -1415,6 +1416,29 @@ def test_i18n_patterns_returns_list(self):
         with override_settings(USE_I18N=True):
             self.assertIsInstance(i18n_patterns([]), list)
 
+    @override_settings(
+        LANGUAGE_CODE='pt',
+        LOCALE_PATHS=[
+            os.path.join(here, 'locale'),
+            (os.path.join(here, 'locale'), 'custom_domain')
+        ]
+    )
+    def test_custom_gettext_domains(self):
+        """
+        Locale files with custom names have been successfully loaded.
+
+        The locale files are located in i18n/locale/pt/LC_MESSAGES.
+        """
+        self.assertEqual(gettext('This is a test of the default gettext domain loading.'),
```

Comment:
Use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).